package.json

```diff
@@ -31,6 +29,7 @@
     "jsonwebtoken": "^8.3.0",
     "keycloak-auth-utils": "^3.3.0",
     "lodash.get": "^4.4.2",
+    "log": "^6.0.0",
```

Comment:
@rakeshSgr - Remove cors, log, apidoc etc

@deepakr123 - Please review this package.json completely, keep only required modules.